NotifyTests/IntegrationTests/Assertions.cs

```diff
@@ -0,0 +1,87 @@
+using Notify.Client;
+using Notify.Models;
+using Notify.Models.Responses;
+using Notify.Exceptions;
+using System;
+using System.Collections.Generic;
+
+using NUnit.Framework;
+
+namespace Notify.IntegrationTests.Assertions
+{
+	public class NotifyAssertions
+	{
+		static public void AssertNotification(Notification notification)
+		{
+			Assert.IsNotNull(notification.type);
+			String notificationType = notification.type;
+			String[] allowedNotificationTypes = { "email", "sms", "letter" };
+			CollectionAssert.Contains(allowedNotificationTypes, notificationType);
+			if (notificationType.Equals("sms"))
+			{
+				Assert.IsNotNull(notification.phoneNumber);
+			}
+			else if (notificationType.Equals("email"))
+			{
+				Assert.IsNotNull(notification.emailAddress);
+				Assert.IsNotNull(notification.subject);
+			}
+			else if (notificationType.Equals("letter"))
+			{
+				Assert.IsNotNull(notification.subject);
+			}
```

Comment:
else  Assert.Fail();

Reply:
Not sure this is necessary as it would never hit the else as we have this line before - `CollectionAssert.Contains(allowedNotificationTypes, notificationType);`